### PR TITLE
Refactored to Polars: Chapter 4

### PR DESCRIPTION
**Pull request by jps215**

### CODE REVIEW

File: cookbook/Chapter 4 - Find out on which weekday people bike the most.py [hunk 1]
```diff
@@ -1,97 +1,65 @@
...
 # %% Load the data
-bikes = pd.read_csv(
+bikes = pl.read_csv(
     "../data/bikes.csv",
     sep=";",
     encoding="latin1",
-    parse_dates=["Date"],
-    dayfirst=True,
-    index_col="Date",
+    parse_dates=True,
```
Comment (yuanjingshi) [inline cookbook/Chapter 4 - Find out on which weekday people bike the most.py]: it changes to true_parse_dates in Polars package

File: cookbook/Chapter 4 - Find out on which weekday people bike the most.py [hunk 2]
```diff
@@ -1,97 +1,65 @@
...
-# This is necessary to show lots of columns in pandas 0.12.
-# Not necessary in pandas 0.13.
-pd.set_option("display.width", 5000)
-pd.set_option("display.max_columns", 60)
-
 # %% Load the data
-bikes = pd.read_csv(
+bikes = pl.read_csv(
     "../data/bikes.csv",
     sep=";",
```
Comment (ty338jy) [inline cookbook/Chapter 4 - Find out on which weekday people bike the most.py]: Hey Joe, i think in polars should be separator=";"

File: cookbook/Chapter 4 - Find out on which weekday people bike the most.py (hunk 1 above)
Comment (jps215) [reply cookbook/Chapter 4 - Find out on which weekday people bike the most.py]: Thanks, fixed! :)

File: cookbook/Chapter 4 - Find out on which weekday people bike the most.py (hunk 2 above)
Comment (jps215) [reply cookbook/Chapter 4 - Find out on which weekday people bike the most.py]: Thanks, fixed! :)

**Review by yuanjingshi — APPROVED** [review]
Looks nice!

**Pull request merged**